schema.sql

```diff
@@ -18,14 +18,14 @@ CREATE TABLE owners (
     id SERIAL PRIMARY KEY NOT NULL,
     full_name varchar(100),
     age INT,
-    PRIMARY KEY(owner_id)
+    (owner_id)
```

Comment:
- [OPTIONAL] I think the `id` field is enough to represent a single owner. Adding `(owner_id)` is not relevant again. It will be nice if you could remove it.